scripts/react-app/opener.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+let childProcess = require("child_process");
+let os = require("os");
+
+module.exports = function opener(args, options, callback) {
+    let platform = process.platform;
```

Comment:
would be good to extract this up to line 13 into its own function called `get_platform`

Reply:
Should I make a seperate file for getPlatform func

Reply:
pls see comment on line 5 (i just added it). create one new file called `packageUtil.js` and put all these functions in there 👍